### ISSUE TITLE
[Feature request] open file with root rights (for editing it)

### ISSUE THREAD

**Issue by Golddouble** [cboxdoerfer/fsearch#448]
**Is your feature request related to a problem? Please describe.**
Sometimes I look for a conf file that lies on the root partition in Linux and I want to edit it. It would be user friendly, if we were able to right click on the search result of FSearch and have the entry "edit with root rights with for example Featherpad".

**Describe alternatives you've considered**
Until now I use right click "open folder". Thunar opens. In thunar I look again for the file and choose in thunar "edit as root". Which is a bit cumbersome.

Maybe you can add an option in "settings", so that users, who are afraid of destroying their system by such a possibility, can deselect this possibility. Or it may be by default inactivated, but can be activated in settings.

What do you think?

**Comment by cboxdoerfer** [comment]
Isn't the "Edit as root" action in Thunar implemented as a custom action (`Edit -> Configure custom actions...`)? Having those in FSearch would indeed be great.

**Comment by Golddouble** [comment]
> Isn't the "Edit as root" action in Thunar implemented as a custom action (`Edit -> Configure custom actions...`)? Having those in FSearch would indeed be great.

Yes, it is. But this is less user-friendly. See "**Describe alternatives you've considered**" in Post #1

**Comment by cboxdoerfer** [comment]
Yes, what I meant is that instead of adding a general **Edit as root** action to FSearch, it might be better to just add custom actions in FSearch—and a list of useful custom actions to the documentation. Then users can have all sorts of actions (e.g. **Edit as root**, *Open terminal here**, ...) right within FSearch and FSearch doesn't have to guess which editor, terminal, ... the user prefers for those commands.

**Comment by Golddouble** [comment]
Yes, I think this would be a great solution. So the user could make this "custom action" using the editor he likes most.
